javascript/builtins/RegExp.json

```diff
@@ -91,6 +91,735 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "backreference": {
```

Comment:
Should this be `javascript.builtins.RegExp.RegExp.backreference`, or simply `javascript.regular_expressions.backreference` (i.e. create a new top-level API)? I prefer the latter because it aligns with the content slugs and decouples the regex syntax from the constructor (since there's also the regex literal syntax).

Reply:
I'm fine with the latter also if there are no objections. I'll raise it with the rest of the team.

Reply:
I've opened another draft here so we can get an idea of how this would look like: https://github.com/mdn/browser-compat-data/pull/18966/files

What do you think?

Reply:
I'm going to close this PR in favor of #18966